src/TwilioVideo.ios.js

```diff
@@ -303,6 +340,11 @@ export default class extends Component {
           this.props.onParticipantDisabledAudioTrack(data)
         }
       }),
+      this._eventEmitter.addListener('dataTrackMessageReceived', data => {
+        if (this.props.onDataTrackMessageReceived) {
+          this.props.onDataTrackMessageReceived(data)
```

Comment:
Does this need to unwrap it from the message field of  data?

Reply:
Looks good to me